packages/subgraph/package.json

```diff
@@ -14,7 +14,7 @@
     "deploy": "scripts/deploy-subgraph.sh",
     "create:local": "graph create aragon/aragon-core-goerli --node http://localhost:8020",
     "deploy:local": "LOCAL=true scripts/deploy-subgraph.sh",
-    "generate:introspection": "graphql-codegen && cd docs && prettier schema-introspection.json --write",
+    "generate:introspection": "scripts/generate-introspection-json.sh",
```

Comment:
perhaps we need to create 2 action here one for partial & one for full introsepction